packages/nimble-components/src/wafer-map/specs/README.md

```diff
@@ -0,0 +1,235 @@
+-   Zoom out while hovering - this action gets executed whenever the mouse pointer hovers the `nimble-wafer-map` component and a wheel event (scroll down) gets triggered
+-   Drag while zoomed - this event gets triggered whenever the `nimble-wafer-map` component is zoomed in (larger than 100%) and whilst the left mouse button is held the pointer moves to any direction within the wafer-map canvas
+-   Mouse hover - this event gets triggered whenever the mouse pointer is hovering any of the die elements within the wafer map. We only must detect this in the nimble component, proper handling will be done in the MicroStrategy wrapper. (Tooltip triggering)
+
+### Anatomy
+
+Shadow DOM:
+
+```html
+<template id="WaferMapTemplate">
+    <div class="WaferMapContainer">
+        <svg>
+            <g class="zoomContainer">
+                <g>
+                    <circle></circle>
+                    <circle></circle>
+                    <rect></rect>
+                </g>
+            </g>
+        </svg>
+        <div class="WaferMapArea">
+            <canvas />
+            <svg>
+                <rect></rect>
+            </svg>
+        </div>
+    </div>
+</template>
+```
+
+The first SVG element with the "zoomContainer" class attribute contains two circles and a rectangle.
+
+-   The first circle represents the outline of the wafer map component.
+-   The second circle, together with the rectangle is part of a solution which creates the illusion that there is a notch present on the wafer map frame.
+
+The second SVG element with the "WaferMapArea" class attribute represents the area where the dies get rendered.
+The rectangle present here is used as the "highlight" which allows us to identify individual dies in the canvas.
+
+-   Slot Names: (none)
+-   Host Classes: (none)
+-   Slotted Content/Slotted Classes: (none)
+-   CSS Parts: (none)
+
+### Angular integration
+
+The `nimble-wafer-map` component does not take user input directly from the webpage where it's embedded. All the inputs must be provided through the constructor of the component so in the current case there is no point using ControlValueAccessors.
+
+### Blazor integration
+
+Blazor support is out of scope for the current implementation.
+
+### Visual Appearance
+
+For the initial component, we are going with the original wafer map appearance.
+![WaferMap Example](./Resources/wafer_map_example.png)
+
+## Implementation
+
+As mentioned above, this is not a component that we start from scratch, but a visualization that already exists which is outdated and not futureproof.
+As a first step, we are working on the "bloat removal" which is basically the removal of unwanted but exiting features (legend, tooltip etc.) and dependencies.
+
+Original component: [wafermap](https://dev.azure.com/ni/DevCentral/_git/op-web-portal?path=/libraries/wafermap&version=GBfeature/microServicesSplit)\
+Once the "bloat removal" is finished, the next step is to adapt the current
```

Comment:
Was there more to this sentence? It looks like it is cut-off.

Reply:
@rajsite I missed this one. I added the missing information.